File: Java/src/main/java/com/semantria/mapping/output/DocPhrase.java
package com.semantria.mapping.output;

import javax.xml.bind.annotation.XmlElement;

public class DocPhrase
{
    private String title = null;
    private Float sentiment_score = null;
    private Boolean is_negated = null;
    private String negating_phrase = null;
    private Boolean is_intensified = null;
    private String intensifying_phrase = null;
	private String sentiment_polarity = null;
	private String type = null;

    @XmlElement(name="title")
    public String getTitle() { return title; }
    @XmlElement(name="sentiment_score")
    public Float getSentimentScore() { return sentiment_score; }
    @XmlElement(name="is_negated")
    public Boolean getIsNegated() { return is_negated; }
    @XmlElement(name="negating_phrase")
    public String getNegatingPhrase() { return negating_phrase; }
    @XmlElement(name="is_intensified")
    public Boolean getIsIntensified() { return is_intensified; }
    @XmlElement(name="intensifying_phrase")
    public String getIntensifyingPhrase() { return intensifying_phrase; }
	@XmlElement(name = "sentiment_polarity")
	public String getSentimentPolarity() { return sentiment_polarity; }
	@XmlElement(name = "type")
	public String getType() { return type; }

    public void setTitle(String title) {this.title = title;}
    public void setSentimentScore(Float sentiment_score) {this.sentiment_score = sentiment_score;}
    public void setNegatingPhrase(String negating_phrase) {this.negating_phrase = negating_phrase;}
    public void setIsNegated(Boolean is_negated) {this.is_negated = is_negated;}
    public void setIsIntensified(Boolean is_intensified) {this.is_intensified = is_negated;}
    public void setIntensifyingPhrase(String intensifying_phrase) {this.intensifying_phrase = negating_phrase;}
	public void setSentimentPolarity(String sentiment_polarity) { this.sentiment_polarity = sentiment_polarity; }
	public void setType(String type) { this.type = type; }
}

File: dotNet/Semantria.Com/Mapping/Output/DocPhrase.cs
using System.Runtime.Serialization;
using System.Xml.Serialization;

namespace Semantria.Com.Mapping.Output
{
    [DataContract(Name = "phrase")]
    [XmlRootAttribute("phrase", Namespace = "")]
    public class DocPhrase
    {
        [DataMember(Name = "title")]
        [XmlElementAttribute("title")]
        public string Title { get; set; }

        [DataMember(Name = "type")]
        [XmlElementAttribute("type")]
        public string Type { get; set; }

        [DataMember(Name = "is_negated")]
        [XmlElementAttribute("is_negated")]
        public bool IsNegated { get; set; }

        [DataMember(Name = "negating_phrase")]
        [XmlElementAttribute("negating_phrase")]
        public string NegatingPhrase { get; set; }

        [DataMember(Name = "is_intensified")]
        [XmlElementAttribute("is_intensified")]
        public bool IsIntensified { get; set; }

        [DataMember(Name = "intensifying_phrase")]
        [XmlElementAttribute("intensifying_phrase")]
        public string IntensifyingPhrase { get; set; }

        [DataMember(Name = "sentiment_score")]
        [XmlElementAttribute("sentiment_score")]
        public float SentimentScore { get; set; }

        [DataMember(Name = "sentiment_polarity")]
        [XmlElementAttribute("sentiment_polarity")]
        public string SentimentPolarity { get; set; }

        [DataMember(Name = "offset")]
        [XmlElementAttribute("offset")]
        public string Offset { get; set; }

        [DataMember(Name = "length")]
        [XmlElementAttribute("length")]
        public string Length { get; set; }
    }
}
